boards/arm/nucleo_f767zi/doc/index.rst

```diff
@@ -0,0 +1,226 @@
+.. _nucleo_f767zi_board:
+
+ST Nucleo F767ZI
+################
+
+Overview
+********
+
+The STM32 Nucleo-144 boards offer combinations of performance and power that
```

Comment:
Hmmm... the title is for this doc is``ST Nucleo F767ZI`` board, but you're documenting a different board here: ``STM32 Nucleo-144``.  Something's amiss.

Reply:
Following several other Nucleo document format for this Nucleo-144 series board, such as nucleo_f746zg and nucleo_f756zg. I personally have no preference.

Reply:
Well, that's a bit confusing but I get it now.  The STM32 Nucleo-144 boards are a family while NUCLEO-L496ZG and NUCLEO-L496ZG-P, for example, are specific members of that family of boards.